kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -6,15 +6,15 @@
       class="dropdown-button"
       @click="toggleDropdown"
       aria-haspopup="true">
-      <span class="dropdown-button-text" :class="{uparrow: dropdownopen}">
+      <span class="dropdown-button-text" :class="{'dropdown-button-text-open': dropdownOpen}">
         {{ $tr('downloadContent') }}
       </span>
     </button>
     <ul
       ref="dropdownitems"
       class="dropdown-items"
-      :class="{ dropdownopen: dropdownopen }"
-      :aria-hidden="dropDownOpenText"
+      :class="{ 'dropdown-open': dropdownOpen }"
+      :aria-hidden="dropdownOpenText"
```

Comment:
From his [earlier post](https://www.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/), it sounds like ARIA and HTML5 styles both semantically impact screen reader behavior.:

> In supporting browsers the content is not displayed to any user.

`hidden` is [supported in Android 4+ but not IE10](http://caniuse.com/#feat=hidden). What was the final decision around IE10 support?

cc @radinamatic in case you want to weigh in